examples/notebooks/Validating_mechanical_models_Enertech_DFN.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test the parameter set of the Enertech cells\n",
+    "In this notebook, we show how to use pybamm to reproduce the experimental results for the Enertech cells (LCO-G). To see all of the models and submodels available in PyBaMM, please take a look at the documentation [here](https://pybamm.readthedocs.io/en/latest/source/models/index.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pybamm\n",
```

Comment:
you need to add a line `%pip install pybamm -q    # install PyBaMM if it is not installed` so that this example works in Google Colab

Reply:
Done. Do have a check of scaling and nondimensionalizations. I have checked them for a few times, but there might still be some misunderstanding.